server/routerlicious/packages/services-core/src/mongoDatabaseManager.ts

```diff
@@ -43,7 +45,10 @@ export class MongoDatabaseManager implements IDatabaseManager {
     }
 
     private async getCollection<T>(name: string) {
-        const db = await this.mongoManager.getDatabase();
+        const db = name === this.documentsCollectionName && this.globalDbEnabled ?
+        await this.globalDbMongoManager.getDatabase() :
```

Comment:
I think a tab here at line 49 and another one in line 50 would it make it easier to read this statement. Otherwise, it might confuse somebody reading the code for the first time, who might not realize that the await statements are a continuation of line 48.